src/App.tsx

```diff
@@ -7,48 +7,73 @@ import AuthMenu from './auth0/AuthMenu';
 import Client from './components/client/Client';
 import MainRouter from './components/router/MainRouter';
 import React, { Suspense, useEffect, useState } from 'react';
-import { Link, Route, Routes } from 'react-router-dom';
+import { Link, Route, Routes, useLocation } from 'react-router-dom';
 import BaseDetailsManager from './components/createBusiness/baseDetailsManager';
 import EmailVerification from './components/createBusiness/emailVerification';
 import MoreDetailsManager from './components/createBusiness/moreDetailsManager';
 import { useAppSelector } from './Redux/hooks';
-import ErrorToast from './components/generic/errorMassage';
+import ErrorToast, { showErrorToast } from './components/generic/errorMassage';
+import Login from './components/Login/login';
+
+
 
 const LazyEditProfile = React.lazy(() => import('./auth0/editProfile'));
 
 const App = () => {
 
   const currentUser = useAppSelector((state) => state.currentUserSlice.CurrentUser);
   const [typeUser, setTypeUser] = useState<any | null>(null);
+  const [lastInvalidPath, setLastInvalidPath] = useState<string | null>(null);
+  const location = useLocation();
+  // const [isLoggedIn, setIsLoggedIn] = useState(true);
 
   useEffect(() => {
     if (currentUser) {
+     
       const type = currentUser.employeeDetails.role.type;
       setTypeUser(type);
     }
   }, [currentUser]);
 
+  const ErrorToastRoute = () => {
+    useEffect(() => {
+      if (location.pathname !== lastInvalidPath) {
+        showErrorToast('הדף שאת/ה מחפש/ת אינו נמצא route-הכנס/י ב http://localhost:0000/link/**של עסק linkUID**');
```

Comment:
למה 0000?

Reply:
לא קשור אלי זה היה בבסיס לשנות ולהוריד בל מה שאני לא מבינה?

Reply:
מה שאני הוספתי זה להוריד את    {/* {isRootPath && (
          <>
            {typeUser !== 'manager' && typeUser !== 'admin' && typeUser !== '' && typeUser !== undefined && typeUser !== null ? (
              <Client />
            ) : typeUser === 'manager' || typeUser === 'admin' ? (
              <>
                <MainRouter />
              </>
            ) :  (
                  <Link to="/CreateBusiness/BaseDetailsManager">הרשמה של עסק</Link> ) }
          </>
        )} */}
וגם הוספתי לינק בפרופיל - זהו!